devicetypes/fibargroup/fibaro-walli-double-switch.src/fibaro-walli-double-switch.groovy

```diff
@@ -0,0 +1,533 @@
+/**
+ *  Copyright 2020 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "Fibaro Walli Double Switch", namespace: "fibargroup", author: "SmartThings", mnmn: "SmartThings", vid: "generic-switch-power-energy", genericHandler: "Z-Wave") {
+		capability "Actuator"
+		capability "Configuration"
+		capability "Health Check"
+		capability "Energy Meter"
+		capability "Power Meter"
+		capability "Refresh"
+		capability "Sensor"
+		capability "Switch"
+
+		command "reset"
+
+		fingerprint mfr: "010F", prod: "1B01", model: "1000", deviceJoinName: "Switch" //Fibaro Walli Double Switch FGWDSEU-221, Raw Description zw:Ls type:1001 mfr:010F prod:1B01 model:1000 ver:5.01 zwv:6.02 lib:03 cc:5E,55,98,9F,56,6C,22 sec:25,85,8E,59,86,72,5A,73,32,70,71,75,5B,7A,60 role:05 ff:9D00 ui:9D00 epc:2
```

Comment:
can you move the comment to another line? this DTH is very W I D E

Reply:
ok, fixed